helpers/email/email.py

```diff
@@ -63,5 +63,23 @@ def event_cancellation_notification(
             event_reject_reason=event_reject_reason
         )
 
+    def admin_invite(self, user_email, user_name):
```

Comment:
Consider naming this function as `send_admin_invite_email()` if it's purpose is to send an invitation email to admins or rather use function/method names that clearly indicate what the function does and are a verb in nature like `do_something()`

Reply:
Alright, I will change that.

Reply:
Done